Doc/library/stdtypes.rst

```diff
@@ -949,7 +949,7 @@ Generator Types
 ---------------
 
 Python's :term:`generator`\s provide a convenient way to implement the iterator
-protocol.  If a container object's :meth:`~iterator.__iter__` method is implemented as a
+protocol.  If a container object's :meth:`~object.__iter__` method is implemented as a
 generator, it will automatically return an iterator object (technically, a
 generator object) supplying the :meth:`!__iter__` and :meth:`~generator.__next__`
```

Comment:
For the second one, `__iter__` would not be the `__iter__` from `object` but the `__iter__` from `iterator`.